Simplenote/SPAppDelegate+Extensions.swift

```diff
@@ -394,3 +394,12 @@ extension SPAppDelegate {
         EditorFactory.shared.scrollPositionCache.cleanup(keeping: allIdentifiers)
     }
 }
+
+// MARK: - Core Data
+//
+extension SPAppDelegate {
+    @objc
+    func managedObjectContext() -> NSManagedObjectContext {
```

Comment:
Nipticky Suggestion!

```
@objc
var managedObjectContext: NSManagedObjectContext {
    //...
}
```

That way you get away without changing the way managedObejctContext is invoked elsewhere.